cmd/ndm_daemonset/filter/osdiskexcludefilter_test.go

```diff
@@ -63,24 +63,60 @@ func TestOsDiskFilterRegister(t *testing.T) {
 }
 
 func TestOsDiskExcludeFilterExclude(t *testing.T) {
-	fakeDiskPath := "fake-disk-path"
-	ignoreDiskPath := "ignore-disk-path"
-	fakeOsDiskFilter := oSDiskExcludeFilter{excludeDevPath: ignoreDiskPath}
-	disk1 := &controller.DiskInfo{}
-	disk1.Path = fakeDiskPath
-	disk2 := &controller.DiskInfo{}
-	disk2.Path = ignoreDiskPath
 	tests := map[string]struct {
+		filter   oSDiskExcludeFilter
 		disk     *controller.DiskInfo
-		actual   bool
 		expected bool
 	}{
-		"comparing return of Exclude for disk1": {actual: fakeOsDiskFilter.Exclude(disk1), expected: true},
-		"comparing return of Exclude for disk2": {actual: fakeOsDiskFilter.Exclude(disk2), expected: false},
+		"exclude path is /dev/sda and device path is /dev/sda": {
```

Comment:
Good if we cover "dm" test case and anyother device which we anticipate.

Reply:
Added test case for lvm devices.